File: 0066-plus-one/0066-plus-one.js
/**
 * @param {number[]} digits
 * @return {number[]}
 */
var plusOne = function(digits) {
    let str=digits.toString().replaceAll(",","");
    let num= BigInt(str)
  
    let sum=num+1n;

    let st=sum.toString();

    let res=st.split("");
    return res;
    
};

File: 0066-plus-one/0066-plus-one.cs
public class Solution {
    public int[] PlusOne(int[] digits) {
        string a= String.Join("",digits);
         System.Numerics.BigInteger res = System.Numerics.BigInteger.Parse(a) + 1;
        string f=res.ToString();
        int[] p=new int[f.Length];
        for(int i=0;i<f.Length;i++){
             p[i] = int.Parse(f[i].ToString());
        }
        return p;



        
    }
}